queue_util/__init__.py

```diff
@@ -1,3 +1,26 @@
 from queue_util.consumer import Consumer
 from queue_util.producer import Producer
 from queue_util.producer import get_num_messages
+
+# kombu v4 will come out with the following commit in:
+# https://github.com/celery/kombu/commit/010aae8ccf16ad2fa5a9c3d6f3b84b21e1c1677a
+# which does the same thing, but this also allows us to not have to enable
+# insecure serializers
+import msgpack
+from kombu.serialization import register
+
+
+def pack(s):
+    return msgpack.packb(s, use_bin_type=True)
+
+
+def unpack(s):
+    return msgpack.unpackb(s, encoding='utf-8')
+
+register(
+    'unicode-msgpack',
+    pack,
+    unpack,
+    content_type='application/x-unicode-msgpack',
+    content_encoding='binary'
+)
```

Comment:
As we talked about in person, I think we should move this to a separate module (cleaner) and import it here so we make sure it's run and registered.